wodles/aws/aws_s3.py

```diff
@@ -2765,19 +2787,28 @@ def get_alerts(self):
             last_scan = initial_date
 
         datetime_last_scan = datetime.strptime(last_scan, '%Y-%m-%d %H:%M:%S.%f')
+        date_scan = datetime_last_scan if not self.only_logs_after or datetime_last_scan > \
+            (date_ol := datetime.strptime(self.only_logs_after, "%Y%m%d")) else date_ol
         # get current time (UTC)
         datetime_current = datetime.utcnow()
         # describe_findings only retrieves 100 results per call
         response = self.client.list_findings(maxResults=100, filter={'creationTimeRange':
-                                                                         {'beginDate': datetime_last_scan,
+                                                                         {'beginDate': date_scan,
                                                                           'endDate': datetime_current}})
+        debug(f"+++ Listing findings starting from {date_scan}", 2)
         self.send_describe_findings(response['findingArns'])
         # Iterate if there are more elements
         while 'nextToken' in response:
             response = self.client.list_findings(maxResults=100, nextToken=response['nextToken'],
-                                                 filter={'creationTimeRange': {'beginDate': datetime_last_scan,
+                                                 filter={'creationTimeRange': {'beginDate': date_scan,
```

Comment:
```suggestion
                                                 filter={'creationTimeRange': {'beginDate': datetime_last_scan,
```
Update this in case of renaming the variable again